Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabFocusHandler.cpp

```diff
@@ -35,7 +35,7 @@ namespace AzToolsFramework::Prefab
     {
         InstanceOptionalReference focusedInstance;
 
-        if (entityId == AZ::EntityId())
+        if (entityId == static_cast<AZ::EntityId>(AZ::EntityId::InvalidEntityId))
```

Comment:
This still constructs and `if (!entityId.IsValid())` should be a bit more concise anyways

Reply:
Fair. It is weird that InvalidEntityId is not an AZ::EntityId.